docs/en-US/QUICKSTART.md

```diff
@@ -96,6 +96,12 @@ On you chosen chat service, these are the example messages based on regex_matche
 * `get vmware VMs protected by {sla}`
 * `perform instant recovery of vmware VM {vm_name} from {MM-DD-YYYY} at {HH:MM AM/PM}`
```

Comment:
change to `perform instant recovery of vmware vm {vm_name} from {MM-DD-YYYY} at {HH:MM AM/PM}`